tests/articulation_tests.ts

```diff
@@ -6,11 +6,11 @@
 import { Articulation } from 'articulation';
 import { Beam } from 'beam';
 import { Flow } from 'flow';
-import { Font } from 'font';
+// import { Font } from 'font'; // RONYEH
```

Comment:
You could leave this file unmodified, `Font.SANS_SERIF`is already merged producing no visual diff. And the eslint fixes are included #1209.